catalog-info.yaml

```
@@ -693,3 +693,32 @@ spec:
           access_level: BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-elastic-agent-binary-dra-7-17
+  description: Buildkite pipeline for packaging Elastic Agent core binary and publish it to DRA
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/buildkite-elastic-agent-binary-dra-7-17
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: buildkite-elastic-agent-binary-dra-7-17
+      description: Buildkite pipeline for packaging Elastic Agent core binary and publish it to DRA
+    spec:
+      pipeline_file: ".buildkite/x-pack/elastic-agent/.buildkite/pipeline.xpack.elastic-agent-binary-dra.yml"
```

Should this particular pipeline be filtered for only the `7.17` branch and branches targeting `7.17`?

```suggestion
      pipeline_file: ".buildkite/x-pack/elastic-agent/.buildkite/pipeline.xpack.elastic-agent-binary-dra.yml"
      provider_settings:
        build_branches: true
        cancel_deleted_branch_builds: true
        filter_condition: 'build.branch == "7.17" || build.pull_request.base_branch =~ /7.17/'
        filter_enabled: true
```

According to this [docs](https://registry.terraform.io/providers/buildkite/buildkite/latest/docs/resources/pipeline#provider-settings-configuration) it's possible to use `filter_condition` with further conditions:

> 
build.pull_request.base_branch | String, null | The base branch that the pull request is targeting, otherwise null if the branch is not a pull request
-- | -- | --

---

I don't know if it's also required to configure whether to cancel the builds with `cancel_intermediate_builds`, `cancel_intermediate_builds_branch_filter`, `skip_intermediate_builds` and `skip_intermediate_builds_branch_filter`

---

> Let's hope there will be no 7.18 :)

There will be none. `7.17` is the very last minor for `7`.